### ISSUE TITLE
Bindings for attributes passed from an #each helper to a inner component do not work properly in Ember 1.13+

### ISSUE THREAD

**Issue by exop-it** [emberjs/ember.js#11647]
I have a `inner-component` defined as
```handlebars
<h4>Item {{index}} - {{title}}</h4>
Edit Title: {{input value=title}}
```
which is used in the following way (`items` is just an array of strings):
```handlebars
{{#each items as |item index|}}
  {{inner-component title=item index=index}}
{{/each}}
```
Changing the contents of the text field does not update the `title` property and triggers no observers in the component. Ember versions before 1.13.0 work just fine.

Here's a [JSBin](http://jsbin.com/kuraki/2/edit?html,js,output) which demonstrates this behaviour.

**Comment by stefanpenner** [comment]
I seem to remember a related fix. @tomdale was this the fix you landed earlier this week?

**Comment by stefanpenner** [comment]
@exop-it can you check on canary?

**Comment by rwjblue** [comment]
Fixed in latest release channel builds, and will be released in 1.13.3 soon.

Demo: http://rwjblue.jsbin.com/kuqiru/1/edit?html,js,output

**Comment by stefanpenner** [comment]
@rwjblue :)
